src/blocks/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,131 @@
+import { ReactNode, forwardRef, useState } from 'react';
+import styled, { FlattenSimpleInterpolation } from 'styled-components';
```

Comment:
```suggestion
import { forwardRef, useState } from 'react';
import styled from 'styled-components';
```
remove unused imports